compiler/rustc_session/src/options.rs

```diff
@@ -1089,6 +1089,8 @@ options! {
         "allow the linker to link its default libraries (default: no)"),
     embed_bitcode: bool = (true, parse_bool, [TRACKED],
         "emit bitcode in rlibs (default: yes)"),
+    emit_thin_lto: bool = (true, parse_bool, [TRACKED],
```

Comment:
Can you make this a unstable `-Z` option? As written currently, this option would be insta-stable and it will need to go through the stabilization process.

Reply:
Yea, will change it.